census/src/main/java/io/grpc/census/CensusStatsAccessor.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.census;
+
+import com.google.common.base.Stopwatch;
+import com.google.common.base.Supplier;
+import io.grpc.ClientInterceptor;
+import io.grpc.ServerStreamTracer;
+
+/**
+ * Accessor for getting {@link ClientInterceptor} or {@link ServerStreamTracer.Factory} with
+ * default Census stats implementation.
+ */
+public class CensusStatsAccessor {
```

Comment:
I think we may want to hide these, like we do with the Providers (by making a public class within a package-private class).

Reply:
We may want a direct access from [`grpc-testing` package](https://github.com/grpc/grpc-java/blob/49ad4d12d99fd3cf4906f25dc187f442f021f02a/testing/src/main/java/io/grpc/internal/TestingAccessor.java#L31). What about moving this to `io.grpc.internal` package, then we can have this be package-private?